src/DropDownMenu/DropDownMenu.js

```diff
@@ -11,30 +11,35 @@ import keycode from 'keycode';
 import Events from '../utils/events';
 import IconButton from '../IconButton';
 import propTypes from '../utils/propTypes';
+import rtl from '../utils/rtl';
 
 
 function getStyles(props, context) {
   const {disabled} = props;
   const spacing = context.muiTheme.baseTheme.spacing;
   const palette = context.muiTheme.baseTheme.palette;
   const accentColor = context.muiTheme.dropDownMenu.accentColor;
+  const normalizeDirection = rtl(context.muiTheme) || ((style) => {
```

Comment:
Then I might be misunderstanding.
My understanding was that rtl() is a function which returns a function to apply styling logic if invariantDirection was not set (and returns nothing if invariantDriection is set).
Because icon and label properties contains styling attribute which need to be flipped, I'm applying normalizeDirection to these, which flips attribute values if invariantDirection was not set and leaves intact if it is set.

If I'm missing something, could you clarify a little bit more?